frontend/src/modules/WellLogViewer/view/ReadoutWrapper.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+
+import { InfoItem, ReadoutBox, ReadoutItem } from "@modules/_shared/components/ReadoutBox";
+import { Info } from "@webviz/well-log-viewer/dist/components/InfoTypes";
+import { TemplateTrack } from "@webviz/well-log-viewer/dist/components/WellLogTemplateTypes";
+
+import _ from "lodash";
+
+import { DEFAULT_MAX_VISIBLE_TRACKS } from "../utils/logViewerTemplate";
+
+export type ReadoutWrapperProps = {
+    templateTracks: TemplateTrack[];
+    wellLogReadout: Info[];
+    hide?: boolean;
```

Comment:
Is hide ever used? You seem to already hide it in `SubsurfaceLogViewerWrapper` by not rendering it.

Reply:
Is react mounting expensive? The readout will toggle between show/hide whenever you move the mouse in and out of the log-viewer; is it better to mount/unmount it like I do now, is is the hide prop "more" correct for something that might rapidly show/hide

(Considering the size of the component, I assume the difference is negligible, so mostly just curious what the standard is

Reply:
Unfortunately, I can only give you the unsatisfactory answer: it depends.

Usually, it is not expensive to unmount/mount. However, it depends on the side-effects of a mount/unmount - basically, that what is happening in all the `useEffect`s in the respective component (and all its children). Moreover, you loose all your states of course. On the other hand, what is an alternative to toggling visibility via unmount/mount? You could of course make use of CSS properties (`display: hidden | height: 0` etc.) but all the elements will remain in the DOM-tree adding to its calculation-costs. I think the best practice I can recommend is to use mount/unmount as a default unless you have a good reason not to (mounting expenses and/or state). 

However, if you decide to keep the unmount logic here in the parent, you don't need the `hide` prop and could remove it.